### PR TITLE
add popover v-popover-close directive

### PR DESCRIPTION
**Pull request by baiyaaaaa**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by baiyaaaaa** [comment]
不需要再多维护一个变量

**Comment by baiyaaaaa** [comment]
ping @QingWei-Li

File: examples/docs/zh-CN/popover.md [hunk 1]
```diff
@@ -208,26 +207,15 @@ Popover 的属性与 Tooltip 很类似，它们都是基于`Vue-popper`开发的
...
   placement="top"
-  width="160"
-  v-model="visible2">
+  width="160">
   <p>这是一段内容这是一段内容确定删除吗？</p>
   <div style="text-align: right; margin: 0">
-    <el-button size="mini" type="text" @click="visible2 = false">取消</el-button>
-    <el-button type="primary" size="mini" @click="visible2 = false">确定</el-button>
+    <el-button size="mini" type="text" v-popover-close:popover5>取消</el-button>
+    <el-button type="primary" size="mini" v-popover-close:popover5>确定</el-button>
```
Comment (QingWei-Li) [inline examples/docs/zh-CN/popover.md]: popover directive 我都想去掉了 hhhh

**Comment by QingWei-Li** [comment]
用指令非常奇怪啊

**Comment by baiyaaaaa** [comment]
遍历使用的场景下，每一个popover需要多维护一个开关变量好蛋疼了

**Comment by reverland** [comment]
在遍历popover，然后popover里头又有一个关闭该popover的按钮时确实非常蛋疼，:ref="'popover' + $index"。。。

**Pull request closed without merging**